server/src/main/java/org/opensearch/index/query/FieldMaskingSpanQueryBuilder.java

```diff
@@ -54,7 +54,9 @@
  * @opensearch.internal
  */
 public class FieldMaskingSpanQueryBuilder extends AbstractQueryBuilder<FieldMaskingSpanQueryBuilder> implements SpanQueryBuilder {
-    public static final ParseField SPAN_FIELD_MASKING_FIELD = new ParseField("span_field_masking", "field_masking_span");
+
+    public static final String NAME = "span_field_masking";
+    public static final ParseField SPAN_FIELD_MASKING_FIELD = new ParseField(NAME, "field_masking_span");
```

Comment:
Any specific reason for this refactoring?

Reply:
Need the NAME attribute for my code to work. Also good to follow convention of defining the NAME for all the *QueryBuilders